superset-frontend/src/common/components/Modal.tsx

```diff
@@ -121,12 +137,15 @@ export default function Modal({
       ]
     : footer;
 
+  const modalWidth = width || (responsive ? 'calc(100vw - 24px)' : '600px');
```

Comment:
Yeah... this is definitely in _master_ already. Rebase needed :)

Reply:
Sorry about that, rebased now